src/fragments/lib-v1/auth/native_common/user_attributes/common.mdx

```diff
@@ -46,9 +46,9 @@ import flutter8 from '/src/fragments/lib-v1/auth/flutter/user_attributes/30_conf
 
 <Fragments fragments={{ flutter: flutter8 }} />
 
-## Resend verification code
+## Send user attribute verification code
 
-If the code has expired or the user needs to resend the confirmation code, invoke the resend api as shown below:
+If an attribute needs to be verified while the user is authenticated, invoke the send api as shown below:
```

Comment:
For reviewers: this is a change for an API name change

Reply:
To add context - The API was renamed, and during API BR it was determined that the docs for this were wrong. The API should not be used when a sign up code is expired, it should be used anytime an attribute needs to be verified after the user has signed in. The docs were updated in vCurrent but were never updated in vPrevious.